blocks/callout/callout.js

```diff
@@ -1,3 +1,5 @@
+import { cleanText } from '../../blocks-helpers/textHelpers.js';
```

Comment:
Thought I was going to use this, decided not to in case someone bolds text, but I still might end up with nested paragraphs. Thoughts?

Reply:
Actually we might need to chat about multi-paragraph blocks

Reply:
I'll slack you

Reply:
Oh yes I was doing the spread was in the wrong place!!